src/core/CompUnit/Repository/Installation.pm

```diff
@@ -152,8 +156,8 @@ sub MAIN(:$name is copy, :$auth, :$ver, *@, *%) {
         self!resources-dir;
         self!dist-dir;
         self!bin-dir;
-        if ($version < 1) {
-            $.prefix.child('version').spurt('1');
+        if ($version < 2) {
```

Comment:
This breaks the build when a repo containing installed modules is already at version 1 (which would affect most users):
./perl6-m tools/build/install-core-dist.pl /home/nine/rakudo/install/share/perl6
'/home/nine/rakudo/install/share/perl6/short/6D917C2DF32BB3F8FB6B5F8E529BB779C3DCA6F7' is a directory, cannot do '.open' on a directory
  in block <unit> at tools/build/install-core-dist.pl line 24

Since the repo is already using directories in short/. Better to put the spurt call outside the if. At the end of the upgrade procedure, we will have arrived at version 2 in any case.